Makefile

```diff
@@ -69,7 +69,7 @@ latex/book.pdf: latex/book.tex latex/macros.tex
 	(cd latex && ln -f -s ../www/im/ im)
 	(cd latex && pdflatex book.tex)
 
-www/%.html: book/%.md infra/template.html infra/signup.html infra/filter.lua config.json
+www/%.html: book/%.md infra/template.html infra/signup.html infra/filter.lua config.json src/lab*.py
```

Comment:
What does this do? Also, maybe nothing?

Reply:
The book chapters depend on the source code for the outline. This addition expresses that for this particular rule.